o2 check failing with
/sw/SOURCES/O2Physics/8293-slc7_x86-64/0/Common/TableProducer/PID/pidTOF.cxx:206:31: error: 'const struct o2::tof::eventTimeContainer' has no member named 'eventTime'; did you mean 'mEventTime'?
  206 |             float et = evTime.eventTime;
  
  and other simlar errors.
  Do you need to fix naming also there?

--- a/Detectors/TOF/base/include/TOFBase/EventTimeMaker.h
+++ b/Detectors/TOF/base/include/TOFBase/EventTimeMaker.h
@@ -40,6 +40,14 @@ struct eventTimeContainer {
   std::vector<float> mTrackTimes; /// eventtime provided by a single track
   float mDiamondSpread = 6.f;     /// spread of primary verdex in cm. Used when resetting the container to the default value
 
+  // Aliases
+  const float& eventTime = mEventTime;
+  const float& eventTimeError = mEventTimeError;
+  const unsigned short& eventTimeMultiplicity = mEventTimeMultiplicity;
+  const float& sumweights = mSumOfWeights;
+  const std::vector<float>& weights = mWeights;
+  const std::vector<float>& tracktime = mTrackTimes;
+
   void reset()
   {
     // reset info

